app/frontend/components/tab_search/tab_search.js

```diff
@@ -308,9 +343,14 @@ class TabSearch {
     
     var searchHl = $(verseElements).find('.search-hl, .current-hl');
     for (var i = 0; i < searchHl.length; i++) {
-      var highlightedText = $(searchHl[i]);
-      var text = highlightedText.text();
-      highlightedText.replaceWith(text);
+      var highlightedText = $(searchHl[i])[0];
+      var text = document.createTextNode(highlightedText.innerText);
```

Comment:
Let's do let/const inside the block scope. The same is with `for (let i...`

Reply:
I made some adjustments just now!